src/index.css

```diff
@@ -168,6 +168,7 @@ padding:5px; */
 
 .flight-popup {
     background: rgba(90, 90, 90, 0.5);
+    backdrop-filter: blur(1.5px);
```

Comment:
@opheliaprillard22 This is perhaps a controversial change. I put it there, but be ready to revert it. I think it's better with it though.